src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/coreperson/CorePersonService.kt

```diff
@@ -44,6 +44,7 @@ class CorePersonService(
           ethnicity = a.ethnicity?.toCodeDescription(),
           sex = a.gender.toCodeDescription(),
           nameType = a.nameType?.toCodeDescription(),
+          createDate = a.createDate,
```

Comment:
`createDate` already existed on the JPA offender, so just need to return it

Reply:
Do we need it - or just returning because it exists?